src/controllers/index.js

```
@@ -33,12 +44,31 @@ router.get("/:item-:type", (req, res) => {
   } = req.params;
   // data has to be defined as an empty array
   let data = []; //DO NOT CHANGE
-  res.render(type, {
-    type,
-    item,
-    data
-  });
-});
+  if (item === 'dish') {
+    getDishListings()
+      .then(response => {
+        res.render(type, {
+          type,
+          item,
+          data: response
+        });
+      })
+      .catch(err => console.log(err))
+  } else if (item === 'community') {
+    getCommunityListings()
+      .then(response => {
+        console.log(response)
+        res.render(type, {
+          type,
+          item,
+          data: response
+        });
+      })
+      .catch(err => console.log(err))
+  } else {
+    res.send('Error 500')
```

I think this should return a 404 error.

---

done!